types/lightpick/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

Comment:
It looks like the underlying library is UMD, so you might want to have a look at this template: https://github.com/microsoft/TypeScript-Handbook/blob/master/pages/declaration%20files/templates/module-class.d.ts.md

Reply:
Thanks. I've made adjustments according to that template and type definitions for Lodash and jQuery.

I'm not entirely sure though, why you put a comment against tsconfig.json . Should I change anything here? Configs for other UMD types look similar...

Reply:
Sorry, I blanked.  I was thinking that the module type should be "umd", but that doesn't matter for .d.ts files.